### ISSUE TITLE
v0.27.0: Error when missing _geo field

### ISSUE THREAD

**Issue by martinmaillard** [meilisearch/meilisearch#2385]
**Describe the bug**

I just upgraded to 0.27.0rc3 and I now get a `invalid_geo_field` error when indexing documents without a `_geo` field (I don't have this data for all my documents). This was previously working in 0.26.0.

**To Reproduce**
1. Create an index with `_geo` in `filterableAttributes` and `sortableAttributes`
2. Index a document without a `_geo` field
3. The `documentAddition` task fails with error `invalid_geo_field`

**Expected behavior**
The document should be indexed properly

**Meilisearch version:** v0.27.0rc3

**Additional context**
I'm deploying Meilisearch with docker

**Comment by curquiza** [comment]
Hello @martinmaillard 
Thanks a lot for this report! Sorry we have released v0.27.0, but we will patch the version with a v0.27.1 in the coming week or next week (the teams are currently in offsite)

I succeeded to reproduce

- add this documents
```json
[
  { "id": 2,    "title": "Pride and Prejudice",                    "author": "Jane Austin",              "genre": "romance",    "price": 3.5, "_geo": { "lng": 0, "lat": 0 } },
  { "id": 456,  "title": "Le Petit Prince",                        "author": "Antoine de Saint-Exupéry", "genre": "adventure" , "price": 10.0 }
]
```
- add these settings
```json
{
    "filterableAttributes": ["_geo"],
    "sortableAttributes": ["_geo"]
}
```
- check the `/tasks`
```json
{
    "results": [
        {
            "uid": 1,
            "indexUid": "indexUID",
            "status": "failed",
            "type": "settingsUpdate",
            "details": {
                "filterableAttributes": [
                    "_geo"
                ],
                "sortableAttributes": [
                    "_geo"
                ]
            },
            "error": {
                "message": "The document with the id: `456` contains an invalid `_geo` field.",
                "code": "invalid_geo_field",
                "type": "invalid_request",
                "link": "https://docs.meilisearch.com/errors#invalid_geo_field"
            },
            "duration": "PT0.015387700S",
            "enqueuedAt": "2022-05-09T13:38:14.1008224Z",
            "startedAt": "2022-05-09T13:38:14.106651Z",
            "finishedAt": "2022-05-09T13:38:14.1220387Z"
        },
        {
            "uid": 0,
            "indexUid": "indexUID",
            "status": "succeeded",
            "type": "documentAddition",
            "details": {
                "receivedDocuments": 2,
                "indexedDocuments": 2
            },
            "duration": "PT0.130494100S",
            "enqueuedAt": "2022-05-09T13:38:01.0173966Z",
            "startedAt": "2022-05-09T13:38:01.0237828Z",
            "finishedAt": "2022-05-09T13:38:01.1542769Z"
        }
    ]
}
```

Thanks again @martinmaillard this is really helpful!! 🙏 

🔥 We need to add tests! 🔥

**Comment by curquiza** [comment]
Fixed by #2387, will be released in v0.27.1
